File: sandbox/python/src/TemplateUpdateFilesDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    template_update_files_request = models.TemplateUpdateFilesRequest(
        file_urls=[
            "https://www.dropbox.com/s/ad9qnhbrjjn64tu/mutual-NDA-example.pdf?dl=1",
        ],
    )

    try:
        response = api.TemplateApi(api_client).template_update_files(
            template_id="f57db65d3f933b5316d398057a36176831451a35",
            template_update_files_request=template_update_files_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Template#template_update_files: %s\n" % e)

File: sandbox/dotnet/src/Dropbox.SignSandbox/TemplateUpdateFilesDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TemplateUpdateFilesDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        var templateUpdateFilesRequest = new TemplateUpdateFilesRequest(
            fileUrls: [
                "https://www.dropbox.com/s/ad9qnhbrjjn64tu/mutual-NDA-example.pdf?dl=1",
            ]
        );

        try
        {
            var response = new TemplateApi(config).TemplateUpdateFiles(
                templateId: "f57db65d3f933b5316d398057a36176831451a35",
                templateUpdateFilesRequest: templateUpdateFilesRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Template#TemplateUpdateFiles: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
